src/lib/support/JniReferences.h

```diff
@@ -160,6 +160,12 @@ class JniReferences
         return CreateBoxedObject(boxedTypeClsName, constructorSignature, chip::to_underlying(value), outObj);
     }
 
+    /**
+     * Use instead of 'NewStringUTF' function
+     * Creates a java string type based on char array.
```

Comment:
Please explain why to use instead of NewStringUTF: what are the differences? 

As far as I can tell, this is using Charset(UTF-8)::decode which is lenient to malformed input and replaces it with the charset default replacement byte array, while NewStringUTF is strict.

Based on this description, we can be string on char strings.

Reply:
Yes, I understand what you mean. This part will be corrected when the final fix point is decided.

Reply:
I added some comments.

Reply:
comment has been added.